lib/src/server/call.dart

```diff
@@ -44,6 +44,9 @@ abstract class ServiceCall {
   /// Returns the client certificate if it is requested and available
   X509Certificate? get clientCertificate;
 
+  /// Returns the IP address of the client, if available
+  InternetAddress? get clientIpAddress;
```

Comment:
Could you rename this into `remoteAddress` to match `dart:io` `Socket` API naming?

Reply:
✅ 😄